### ISSUE TITLE
feature request: make board title link to board note

### ISSUE THREAD

**Issue by amandamcg** [joplin/plugin-kanban#23]
Request: When the kanban board is in view, make the title link to the kanban note. This would make it easier to find in long lists.

**Comment by VikramDigra** [comment]
i want to work on this issue please assign me.

**Comment by PackElend** [comment]
@VikramDigra
 No need to asign anyone, just said you want to work on this so simply do it 🙂

**Comment by DarkPheonix12** [comment]
Hey, Is this issue still open?

**Comment by PackElend** [comment]
yes but the original maintainer left. Feel free to play around, trying to maintain it

**Comment by DarkPheonix12** [comment]
Okay. Can you suggest some features that I can contribute.
